### PR TITLE
Use static password in the Hot Rod endpoint

### PR DESCRIPTION
**Pull request by pruivo**
2 changes included in the PR:
* Removed `KC_ISPN_PORT` since it is not possible to change it. 
* Static password (shared password) set for the Hot Rod Endpoint

### CODE REVIEW

File: provision/openshift/Taskfile.yaml [hunk 1]
```diff
@@ -198,19 +195,16 @@ tasks:
         if [ "{{.KC_ISPN_NAMESPACE}}" != "" ]; then
         echo "true" > .task/remote-store-enabled
         echo "infinispan.{{.KC_ISPN_NAMESPACE}}.svc" > .task/remote-store-host
-        oc -n {{.KC_ISPN_NAMESPACE}} get secrets infinispan-generated-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].username > .task/remote-store-username
-        oc -n {{.KC_ISPN_NAMESPACE}} get secrets infinispan-generated-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].password > .task/remote-store-password
+        oc -n {{.KC_ISPN_NAMESPACE}} get secrets connect-secret -o 'jsonpath={.data.identities\.yaml}' | base64 -d | yq .credentials[0].password > .task/remote-store-password
```
Comment (pruivo) [inline provision/openshift/Taskfile.yaml:198]: Should I keep fetching the password from the secret or convert this toa variable?

Comment (kami619) [reply provision/openshift/Taskfile.yaml:198]: is this a secret generated by the operator ? @pruivo how much of an effort would it be to fetch it from the IAM manager ? `--secret-id "$KEYCLOAK_MASTER_PASSWORD_SECRET_NAME"`

File: provision/infinispan/ispn-helm/templates/infinispan.yaml [hunk 2]
```diff
@@ -1,4 +1,13 @@
 {{- $namespace := .Values.namespace | required ".Values.namespace is required." -}}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: connect-secret
```
Comment (pruivo) [inline provision/infinispan/ispn-helm/templates/infinispan.yaml:6]: @kami619 the secret is generated here with the password from AWS.

Comment (kami619) [reply provision/infinispan/ispn-helm/templates/infinispan.yaml:6]: great, thanks

Comment (kami619) [reply provision/infinispan/ispn-helm/templates/infinispan.yaml:6]: in this case we don't need it to be a variable.

**Review by kami619 — APPROVED** [review]
lgtm

**Pull request merged**